test/dnsimple/contacts.spec.js

```diff
@@ -198,6 +198,28 @@ describe('contacts', () => {
         done(error);
       });
     });
+
+    it('includes validation errors coming from the API', async () => {
+      const fixture = testUtils.fixture('createContact/error-validation-errors.http');
+
+      nock('https://api.dnsimple.com')
+        .post('/v2/1010/contacts', attributes)
+        .reply(fixture.statusCode, fixture.body);
+
+      return dnsimple.contacts.createContact(accountId, attributes).then(() => {
+        throw new Error('The promise should follow the rejection path');
+      }, (error) => {
+        expect(error.errors.address1).to.deep.eq(["can't be blank"]);
+        expect(error.errors.city).to.deep.eq(["can't be blank"]);
+        expect(error.errors.country).to.deep.eq(["can't be blank"]);
+        expect(error.errors.email).to.deep.eq(["can't be blank", 'is an invalid email address']);
+        expect(error.errors.first_name).to.deep.eq(["can't be blank"]);
+        expect(error.errors.last_name).to.deep.eq(["can't be blank"]);
+        expect(error.errors.phone).to.deep.eq(["can't be blank", 'is probably not a phone number']);
+        expect(error.errors.postal_code).to.deep.eq(["can't be blank"]);
+        expect(error.errors.state_province).to.deep.eq(["can't be blank"]);
```

Comment:
😅 guess we need to update this to the new requirement, and this should be `error.attribute_errors.XXXXX`...

Reply:
NodeJS is `camelCase` territory, though :)

Reply:
True that 😇 Sorry